src/screens/ProfileScreen/index.js

```diff
@@ -348,7 +363,7 @@
   const goToFollowers = (userId, username) => {
     navigation.navigate('Followers', {
       screen: 'TabFollowing',
-      params: {user_id: userId, username, isFollower: true}
+      params: {user_id: userId, username, isFollower: true, initialData: initialFollowerData}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The navigation parameters now include `initialData` which is set to `initialFollowerData`. This allows the follower data to be passed to the 'Followers' screen.
```diff
-      params: {user_id: userId, username, isFollower: true}
+      params: {user_id: userId, username, isFollower: true, initialData: initialFollowerData}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->